bika/lims/content/sample.py

```diff
@@ -828,7 +910,7 @@ def getLastARNumber(self):
         ar_ids = sorted([AR.id for AR in ARs if AR.id.startswith(prefix)])
         try:
             last_ar_number = int(ar_ids[-1].split("-R")[-1])
-        except:
+        except (IndexError, TypeError, ValueError):
```

Comment:
This is actually nice, but you might introduce with that cosmetic change unexpected errors.
E.g. if `ar_ids[-1]` returns `None` you will get an `AttributeError` here.

Try to keep the focus more on the task to do please.